msm/src/main.rs

```diff
@@ -41,7 +42,7 @@ pub fn main() {
 
     let circuit_env = generate_random_msm_witness();
     let proof_inputs = circuit_env.get_witness();
-    let constraint_exprs = circuit_env.get_exprs_add();
```

Comment:
For a follow-up: we can move the main file in the subdirectory I think

Reply:
Will do